index.js

```diff
@@ -32,24 +25,31 @@ var pify = module.exports = function (fn, P, opts) {
 	};
 };
 
-pify.all = function (obj, P, opts) {
+var pify = module.exports = function (obj, P, opts) {
 	if (typeof P !== 'function') {
 		opts = P;
 		P = Promise;
 	}
 
 	opts = opts || {};
+	opts.exclude = opts.exclude || [/^.+Sync$/];
```

Comment:
> opts.exclude = opts.exclude || [/^.+Sync$/];

Is this a breaking change? It kinda is, but I doubt it changes anything for anyone.